msp/msp_exp_handler.c

```diff
@@ -621,11 +621,28 @@ void msp_exprecv_complete(unsigned char opcode)
 			citiroc_send_slow_control();
 			break;
 
-		case MSP_OP_SEND_CUBES_PROB_CONF:
-			mem_ram_write(RAM_CITI_PROBE, recv_data);
-			citiroc_send_probes();
+		case MSP_OP_SEND_NVM_CITI_CONF:
+			mem_nvm_write(NVM_CITIROC, recv_data);
 			break;
 
+		case MSP_OP_SELECT_NVM_CITI_CONF:
+		{
+			uint8_t cfg_id = (uint8_t)recv_data[0];
```

Comment:
This should be the global variable defined in `main.c`; see my comment on the `cfg_id` variable in `main.c`.